src/tree_sortable.rs

```diff
@@ -0,0 +1,131 @@
+// This file was generated by gir (17af302) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use SortType;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use auto::traits::TreeSortableExt as Auto;
+use std::cell::RefCell;
+use std::mem::transmute;
+use std::cmp::Ordering;
+
+use signal::CallbackGuard;
+
+use glib_ffi::gpointer;
+use {TreeSortable, TreeIter};
+use ffi::{GtkTreeIter, GtkTreeSortable};
+
+pub enum SortColumn {
+    Default,
+    Unsorted,
+    Column(u32),
```

Comment:
Considering there already is "Column" in the enum name, maybe `Id(u32)`? That would rhyme with the method names.

Alternatively we could name this variant `Index` and drop `_id` from the methods. Then the method names would nicely match the type.